packages/lodestar-validator/src/api/impl/rest/beacon/beacon.ts

```diff
@@ -3,11 +3,13 @@ import {BeaconBlock, BeaconState, BLSPubkey, Genesis, ValidatorResponse} from "@
 import {ILogger} from "@chainsafe/lodestar-utils";
 import {Json, toHexString} from "@chainsafe/ssz";
 import {HttpClient, urlJoin} from "../../../../util";
-import {IBeaconApi, IBeaconStateApi} from "../../../interface/beacon";
+import {IBeaconApi, IBeaconBlocksApi, IBeaconStateApi} from "../../../interface/beacon";
+import {RestBeaconBlocksApi} from "./blocks";
 import {RestBeaconStateApi} from "./state";
 
 export class RestBeaconApi implements IBeaconApi {
   public readonly state: IBeaconStateApi;
+  public readonly blocks: IBeaconBlocksApi;
 
   private readonly client: HttpClient;
```

Comment:
seems like we don't use this atm. Do we want to reserve it for any future use?

Reply:
nah, I'm gonna remove it